web/src/store/v1/tag.ts

```diff
@@ -20,12 +23,21 @@ export const useTagStore = create(
         .sort((a, b) => b[1] - a[1])
         .map(([tag]) => tag);
     },
-    fetchTags: async (filter?: string, options?: { skipCache: boolean }) => {
+    fetchTags: async (options: { skipCache?: boolean; user?: User; location?: Location<any> }) => {
```

Comment:
It's better to split it into two args.

```suggestion
    fetchTags: async (params?: {user?: User; location?: Location<any> }, options: { skipCache?: boolean }) => {
```

Reply:
Yeah good shout.